ChangeLog.md

```diff
@@ -1,4 +1,13 @@
-﻿## 2017.09.25 - Version 4.4.0
```

Comment:
@shutchings please revert the changes made to this file, we will handle updating the ChangeLog in the root of the repository during the release

Reply:
Sounds good!